### ISSUE TITLE
Add a blurb for which versions of Vue 2 (2.6 vs 2.7) and Vue 3 are supported/unsupported

### ISSUE THREAD

**Issue by JessicaSachs** [vue-styleguidist/vue-styleguidist#1683]
Hello 👋🏻! I was wondering if I could get a set of version numbers/packages that Vue Styleguidist supports? I'm trying to help someone debug if Vue 2.6 vs 2.7 vs 3.0 is having a particular issue (with Styleguidist's Vue CLI Plugin not being able to find + register components).

I'm reading through the Peer Dependencies for the latest version of Styleguidist and it says it works with any version of Vue 2+, but it requires the SFC Compiler from Vue 3? Is that correct?

Basically, a quick blurb for what versions work with what would be super helpful.

**Comment by elevatebart** [comment]
Hey Jess, 

yeah I will need to get a compat table together.
right now it's only tested with whatever is the latest vue2 version and vue3 version at build time ... on Webpack 4.

The "useful" version of the matrix might be a more complicated multidimensional table than just Vue versions though.

Styleguidist/vue-docgen on its own
Webpack 4/Webpack 5
Vue 2/Vue 3
...Other Libraries
